README.md

```diff
@@ -33,16 +33,17 @@ https://github.com/Superflows-AI/superflows/assets/33871096/3cc09611-a358-4a04-b
 ## Features
 
 - [x] Calls API endpoints to complete tasks and answer questions for users
+- [x] Writes code to analyse data and product visualisations 📊
 - [x] Uses RAG (retrieval) to answer questions on static knowledge 📚
 - [x] Developer dashboard to configure and test your AI assistant 🎛️
 - [x] Stateful streaming API 🏞️
 - [x] UI components (React components [here](https://github.com/Superflows-AI/chat-ui)) 🖥️
 - [x] Upload API specifications for fast set up 💨
 - [x] Asks for user confirmation before taking potentially damaging actions ✅ 
 - [x] Track usage in dashboard 📊
-- [x] Multi-LLM support (Llama 2, GPT4, Claude 2, PaLM 2...) 🌍
+- [x] Multi-LLM support (Finetuned GPT-3.5, GPT-4, Mixtral, Llama2...) 🌍
 - [x] Collects user feedback on AI replies for fine-tuning 👍
-- [x] Self-hosting (including [OS fine-tuned model based on Mistral](https://huggingface.co/Superflows/Superflows-1/tree/main)) 🏠
+- [x] Self-hosting, including with fully open source model Mixtral 🏠
```

Comment:
```suggestion
- [x] Self-hosting, including with fully open source model. **[Reach out](mailto:henry@superflows.ai?subject=Self-hosting+OS+Model:+Superflows&body=Hi+Henry%2C%0A%0AI+work+at+COMPANY+as+ROLE.%0A%0AWe%27d+specifically+like+to+use+the+Open+Source+model+because+REASON.%0A%0AAll+the+best%2C%0AYOUR+NAME+%3A%29) if interested.** 🏠
```